<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataBaseView</class>
 <widget class="QDialog" name="DataBaseView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QScrollBar::vertical
{
	border: none;
	background: rgb(40, 40 , 40);
	width: 8px;
	margin: 10px 0 10px 0;
}

QScrollBar::handle:vertical 
{
	background: rgb(55, 55 , 55);
	border-radius: 4px;
	min-height: 10px;
}

QScrollBar::handle:vertical:hover
{
	background: rgb(77, 230, 230);
}

QScrollBar::handle:vertical:pressed
{
	background:rgb(67, 200, 200);
}

QScrollBar::add-line:vertical 
{
	border: none;
	border-bottom-left-radius: 4px;
	border-bottom-right-radius: 4px;
	background: rgb(40, 40 , 40);
	height: 10px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
  }

  QScrollBar::sub-line:vertical {
	border: none;
	border-top-left-radius: 4px;
	border-top-right-radius: 4px;
	background: rgb(40, 40 , 40);
	height: 10px;
	subcontrol-position: top;
	subcontrol-origin: margin;
  }

QScrollBar::sub-line:vertical:hover
{
	background: rgb(77, 230, 230);
}

QScrollBar::sub-line:vertical:pressed
{
	background: rgb(67, 200, 200);
}

QScrollBar::add-line:vertical:hover
{
	background: rgb(77, 230, 230);
}

QScrollBar::add-line:vertical:pressed
{
	background: rgb(67, 200, 200);
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
{
	background: none;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QFrame" name="Main_frame">
     <property name="styleSheet">
      <string notr="true">QFrame
{
	background-color: rgb(30, 30, 30);
	border-radius: 10px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="Title_bar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="Title_frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);
font-family: 'Roboto Regular';
font-size: 20px;</string>
              </property>
              <property name="text">
               <string>Users data base</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="Buttons_frame">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="hideButton">
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	border: none;
	border-radius: 8px;
	background-color: rgb(85, 255, 127);
}

QPushButton:hover
{
	border: none;
	border-radius: 8px;
	background-color: rgb(64, 191, 93);
}

QPushButton:pressed
{
	border: none;
	border-radius: 8px;
	background-color: rgb(55, 166, 81);
}
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="maximizeButton">
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	border: none;
	border-radius: 8px;
	background-color: rgb(255, 170, 0);
}

QPushButton:hover
{
	border: none;
	border-radius: 8px;
	background-color: rgb(197, 131, 0);
}

QPushButton:pressed
{
	border: none;
	border-radius: 8px;
	background-color: rgb(158, 105, 0)
}
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="closeButton">
              <property name="minimumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16</width>
                <height>16</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	border: none;
	border-radius: 8px;
	background-color: rgb(230, 0, 0);
}

QPushButton:hover
{
	border: none;
	border-radius: 8px;
	background-color: rgb(200, 0, 0);
}

QPushButton:pressed
{
	border: none;
	border-radius: 8px;
	background-color: rgb(160, 0, 0);
}
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="Content_bar">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QTableView" name="DataBaseView_2">
           <property name="styleSheet">
            <string notr="true">QTableView 
{
	background-color: rgb(35, 35 , 35);
	color: rgb(255, 255, 255);
	border: 2px solid rgb(25, 25, 25);
	border-radius: 10px;
	font-family: 'Roboto Regular';
}

QHeaderView::section:horizontal 
{
    color: rgb(255, 255, 255);
    border: 1px solid rgb(31, 31, 31);
    background-color: rgb(35, 35 , 35);
	font-family: 'Roboto Regular';
 }

QHeaderView::section:vertical
{
    color: rgb(255, 255, 255);
    border: 1px solid rgb(31, 31, 31);
    background-color: rgb(35, 35 , 35);
	font-family: 'Roboto Regular';
 }

QTableView QTableCornerButton::section
{
    color: rgb(255, 255, 255);
    border: 1px solid rgb(31, 31, 31);
	border-radius: 10px;
    background-color: rgb(35, 35 , 35);
	font-family: 'Roboto Regular';
 }</string>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
